### PR TITLE
Update and rename pytorch_avoid_softmax_with_nllloss.py to pytorch_av…

### PR DESCRIPTION
**Pull request by grl-wang**
…oid_softmax_with_nllloss_rule.py

https://docs.aws.amazon.com/codeguru/detector-library/python/pytorch-avoid-softmax-with-nllloss-rule/

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by san2488 — APPROVED** [review]
Approving for now, but we should not have the rule suffix
